2_front_end

```diff
@@ -32,6 +32,7 @@ function run_container {
                --build-arg GROUP_ID=$(id -g ${USER}) \
```

Comment:
From what I'm seeing here, you renamed the run_front_end and run_middle_layer scripts, but it would be better to copy them and then ignore your versions. The run_front_end and run_middle_layer scripts are the ones that I use (and that we will tell other people to use when we release it to the public).